src/api/Alert/index.ts

```diff
@@ -370,4 +371,16 @@ export class AlertAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Assign cases from different alerts in bulk to a given user.
+   */
+  public bulkAssignCases(params: IBulkAssignCasesParams) {
```

Comment:
It could make sense to locate that function next to `assignCases(params: IAssignCasesParams)`. These are close relatives.